components/net/src/routing.rs

```diff
@@ -97,8 +97,9 @@ impl BrokerConn {
                 }
                 match parse_from_bytes::<R>(rep.get_body()) {
                     Ok(entity) => Ok(entity),
-                    Err(_) => {
-                        unreachable!("unexpected response, message_id={}", rep.get_message_id())
+                    Err(err) => {
+                        error!("bad route reply, err={}, reply={:?}", err, rep);
```

Comment:
It's the *actual* contents of the message the socket received. Prior I was just printing the message_id in the unreachable code message. That clearly wasn't enough since we *were* expecting a Session back but the message decode *still failed*! :)